src/feedback.ts

```diff
@@ -0,0 +1,28 @@
+import { addComment, closeIssue, fetchOpenIssuesWithLabel } from "./github.ts";
+
+export const run = async () => {
+  // get all issues with the label "status/needs-feedback"
+  const issuesWithStatusNeedsFeedback = await fetchOpenIssuesWithLabel(
+    "status/needs-feedback",
+  );
+  return Promise.all(issuesWithStatusNeedsFeedback.items.map(handleIssue));
+};
+
+// close issue if a month has passed since it was last updated
+const handleIssue = async (issue: {
+  number: number;
+  updated_at: string;
+}) => {
+  const oneMonthAgo = (new Date(Date.now() - 1000 * 60 * 60 * 24 * 30))
+    .getTime();
+  if ((new Date(issue.updated_at)).getTime() < oneMonthAgo) {
+    console.log(`Closing issue #${issue.number} due to feedback timeout`);
+    await addComment(
+      issue.number,
+      `We close issues that need feedback from the author if there were no new comments for a month. :tea:`,
```

Comment:
This seems somewhat unfriendly imo.

> This issue has the `status/needs-feedback` label and hasn't received activity in a month or more, so it is being closed. If you feel this was in error (misplaced label or otherwise), please let us know so we can re-evaluate. If you are not the original author, consider opening a new issue linking to this one instead.

Or something along those lines? The current wording seems very strong to me.

Reply:
Sure, send a PR to change it